routers/init.go

```diff
@@ -56,10 +57,18 @@ func GlobalInit() {
 	if setting.InstallLock {
 		highlight.NewContext()
 		markup.Init()
-
-		if err := models.NewEngine(migrations.Migrate); err != nil {
-			log.Fatal(4, "Failed to initialize ORM engine: %v", err)
+                // In case of delays connecting to DB, retry connection. Eg, PGSQL in Docker Container on Synolgy
+		log.Info("Beginning ORM engine init...")
+		for i := 0; i < setting.DBConnectRetries; i++ {
+			if err := models.NewEngine(migrations.Migrate); err == nil {
+				break
+			} else if i == setting.DBConnectRetries-1 {
+				log.Fatal(4, "Failed to initialize ORM engine: %v", err)
```

Comment:
OK remove this, stick the `if` around the `time.Sleep`, and after the loop return